File: Servidor/InHomeTFGCarlosHernandez/src/main/java/org/tfg/inhometfgcarloshernandez/spring/model/response/GetUsuariosResponseDTO.java
package org.tfg.inhometfgcarloshernandez.spring.model.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.tfg.inhometfgcarloshernandez.domain.model.Usuario;
import org.tfg.inhometfgcarloshernandez.spring.model.UsuarioDTO;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class GetUsuariosResponseDTO {
    private List<UsuarioDTO> usuarios;
}

File: Cliente/InHomeTFGAndroidCarlosHernandez/app/src/main/java/com/example/inhometfgandroidcarloshernandez/data/model/response/GetUsuariosResponseDTO.kt
package com.example.inhometfgandroidcarloshernandez.data.model.response

import com.example.inhometfgandroidcarloshernandez.ui.framework.screens.inmuebles.InmueblesContract

data class GetUsuariosResponseDTO (
    val usuarios: List<InmueblesContract.Usuario>
)